cves/CVE-2013-4315.yml

```diff
@@ -266,7 +307,11 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This vulnerability was the result of not following proper secure coding practices,
+    this includes not sanitizing or checking user input and also most likely assuming 
```

Comment:
"...including not sanitizing, checking user input or assuming..."

Reply:
Changed.